src/constants/enums.js

```diff
@@ -94,6 +100,16 @@ export const AcknowledgementErrorCode = {
     internalErrorCode: `${Gp2gpError.CODE_06.errorCode}-B`,
     internalErrorDescription: 'Partial ingestion, cannot send a full EHR out'
```

Comment:
Should these messages also be parameterised?

Reply:
To me, this isn't something that'd ever be used outside of the context of this object, so I didn't really want to break it out any further.